### PR TITLE
Allow to start additional localstack services for dev services

### PR DESCRIPTION
**Pull request by scrocquesel**
Refs: https://github.com/quarkiverse/quarkus-amazon-services/issues/370

### CODE REVIEW

File: docs/modules/ROOT/pages/includes/quarkus-amazon-kms.adoc [hunk 1]
```diff
@@ -92,7 +92,7 @@ ifndef::add-copy-button-to-env-var[]
 Environment variable: `+++QUARKUS_KMS_DEVSERVICES_SERVICE_NAME+++`
 endif::add-copy-button-to-env-var[]
 --|string 
-|`localstack`
+|required icon:exclamation-circle[title=Configuration property is required]
```
Comment (gsmet) [inline docs/modules/ROOT/pages/includes/quarkus-amazon-kms.adoc:95]: Hum. There is something extremely weird here with the config doc being changed that way.

**Review by gsmet — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/includes/quarkus-amazon-kms.adoc (hunk 1 above)
Comment (scrocquesel) [reply docs/modules/ROOT/pages/includes/quarkus-amazon-kms.adoc:95]: Though the same thing and think it was due to a previous bump version

Comment (scrocquesel) [reply docs/modules/ROOT/pages/includes/quarkus-amazon-kms.adoc]: Issue with my vscode I guess. clean up my m2 registry and build outside vscode did revert changes.

**Review by gsmet — APPROVED** [review]
LGTM. I tweaked the documentation a bit.

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**